package.json

```diff
@@ -2,26 +2,22 @@
   "name": "cloudevents-sdk",
   "version": "2.0.2",
   "description": "CloudEvents SDK for JavaScript",
-  "main": "index.js",
+  "main": "dist/index.js",
   "scripts": {
     "watch": "tsc --project tsconfig.json --watch",
     "build": "tsc --project tsconfig.json && tsc --project tsconfig.browser.json && webpack",
-    "prelint": "npm run build",
-    "lint": "standardx src examples",
-    "fix": "standardx --fix",
-    "pretest": "npm run lint && npm run test:ts",
-    "test": "mocha test/**/*.js",
-    "test:ts": "mocha --require ts-node/register ./test-ts/**/*.ts",
+    "lint": "eslint 'src/**/*.{js,ts}' 'test/**/*.{js,ts}' --fix",
```

Comment:
Using the `--fix` flag might be ok,  but if someone submits a PR and doesn't do the `npm run test` locally first we would never know if there was linting issues since it would just get fixed on CI,  but not committed back

Reply:
Ahh good point.

Reply:
I usually like `npm run fix` along with `npm run lint`.